src/OrganisationUnitTree/checkIfUnitIsSelected.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns true when the path is either in selected
+ * or is a parent path of one of the selected paths
+ *
+ * @param {string} path
+ * @param {string[]} selected
+ * @returns {bool}
+ */
+export const checkIfUnitIsSelected = (path, selected) => selected.includes(path)
```

Comment:
This helper is also used here only, and is just a one-liner, I think it should be removed.